ipfx/feature_vectors.py

```diff
@@ -153,7 +153,7 @@ def feature_vectors(lsq_sweeps, ssq_sweeps, ramp_sweeps,
     """Feature vectors from stimulus set features"""
 
     result = {}
-    result["step_subthresh"] = step_subthreshold(lsq_sweeps, lsq_features, lsq_start, lsq_end)
+    result["step_subthresh"] = step_subthreshold(lsq_sweeps, lsq_features, lsq_start, lsq_end, amp_tolerance=5)
```

Comment:
I have made a PR with some test examples for `isi_shape`, `step_subthreshold` and `subthreshold_depol_norm`. But I cannot decide if these are good or good enough. Could you please review that PR and modify tests as needed. Also, we need tests for the remaining features.

Reply:
I’ll work on the tests (again, I need more information about how to implement them here).